sky/clouds/oci.py

```diff
@@ -295,10 +297,20 @@ def make_deploy_resources_variables(
             cpus=None if cpus is None else float(cpus),
             disk_tier=resources.disk_tier)
 
+        image_tag = self._get_image_str(image_id=resources.image_id,
+                                        instance_type=resources.instance_type,
+                                        region=region.name)
+
+        # pylint: disable=import-outside-toplevel
+        from sky.clouds.service_catalog.oci_catalog import get_image_os_from_tag
+        os_type = get_image_os_from_tag(tag=image_tag, region=region.name)
```

Comment:
style: We import module instead of functions. : )
```suggestion
        from sky.clouds.service_catalog import oci_catalog
        os_type = oci_catalog.get_image_os_from_tag(tag=image_tag, region=region.name)
```